client/app/components/pages/GeneHome.vue

```diff
@@ -853,7 +856,7 @@ export default {
         // backward compatible with old clin.iobio
         "mosaic.chpc.utah.edu":                  {iobio: "mosaic.chpc.utah.edu/gru/api/v1", batchSize: 10},
         "nv-prod.iobio.io":                      {iobio: "mosaic.chpc.utah.edu/gru/api/v1", batchSize: 10},
-        
```

Comment:
This is no biggie, but if you can look at the diffs before a commit, it would be good to eliminate changes that are format related.  Also, I wonder if something is still different with your editor config.

Reply:
Yeah Ill for sure be more careful in the future and double check whats going on with my editor config.  I still seem to be removing any trailing whitespace by default